WebDriverAgentLib/Utilities/FBXCodeCompatibility.m

```diff
@@ -41,5 +46,29 @@ + (instancetype)fb_applicationWithPID:(pid_t)processID
   return [self applicationWithPID:processID];
 }
 
+- (void)fb_activate
+{
+  dispatch_once(&onceActivate, ^{
```

Comment:
You have implemented same thing twice. You probably want to use `fb_isActivateSupported` here

Reply:
fixed, thanks